pkgs/applications/science/chemistry/molden/default.nix

```diff
@@ -19,6 +19,7 @@ stdenv.mkDerivation rec {
                                   --replace 'sudo' "" \
 				                          --replace '-C surf depend' '-C surf'
      sed -in '/^# DO NOT DELETE THIS LINE/q;' surf/Makefile
+     sed -i '/EXTEN.* = exten/d' makefile
```

Comment:
Would it be possible to make a patch for at least `/makefile`? Otherwise it becomes hard to spot what needs to be changed after an update (especially w.r.t. automated updates).

Reply:
OK, I implemented it as patch.